### PR TITLE
add log mount capability

### PR DESCRIPTION
**Pull request by zhangtong007**
test:
test

### CODE REVIEW

File: deploy/helm/lunettes/templates/grafanadi/grafanadi-deploy.yaml [hunk 1]
```diff
@@ -23,6 +23,9 @@ spec:
         - ./grafanadi
         - --config-file=/app/config-file.yaml
         - --enable-kube-client={{ .Values.grafanadiInitKubeClient }}
+        - --log_dir=/logs
```
Comment (linuzb) [inline deploy/helm/lunettes/templates/grafanadi/grafanadi-deploy.yaml:26]: Could you please put these parameters in values.yaml?

Comment (zhangtong007) [reply deploy/helm/lunettes/templates/grafanadi/grafanadi-deploy.yaml]: done

**Review by linuzb — APPROVED** [review]
LGTM

**Review by d3c3mber — APPROVED** [review]
LGTM

**Pull request merged**